GeoMesaQuery.md

```diff
@@ -0,0 +1,81 @@
+# GeoMesa Indexing and Querying #
+
+## The GeoMesa Index ##
```

Comment:
we've got some legacy formats - but I'll just cover the current ones here
All of our formats are in this package: https://github.com/locationtech/geomesa/tree/master/geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/tables